apps/passport-client/components/shared/PCDArgs.tsx

```diff
@@ -368,16 +388,36 @@ function ToggleListArgInput({
     [type]
   );
 
+  const entries = useMemo(
+    () =>
+      showAll
+        ? Object.entries(arg.value)
+        : Object.entries(arg.value).filter(([_, value]) => value),
+    [arg.value, showAll]
+  );
+
   return (
-    <ArgContainer argName={argName} arg={arg}>
-      <ChipsContainer>
-        {Object.entries(arg.value).map(([key, value]) => (
+    <ArgContainer
+      arg={arg}
+      {...rest}
+      end={
+        <ShowMoreButton
+          showAll={showAll}
+          onClick={() => setShowAll((showAll) => !showAll)}
+          size="lg"
+        >
+          {showAll ? "-" : "+"}
```

Comment:
i feel like the icon for showing/hiding all should probably be the same as the one for showing and hiding all the args - rn they're +- and an arrow that turns right or down respectively.